tests/target/string-lit.rs

```diff
@@ -25,5 +25,10 @@ not
 remove
 formatting"#;
 
+    let xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx = funktion("yyyyyyyyyyyyyyyyyyyyy\
+                                                                            yyyyyyyyyyyyyyyyyyyyy\
+                                                                            yyyyyyyyyyyyyyyyyyyyy\
+                                                                            yyyyyyyyyy");
+
```

Comment:
this doesn't look like an improvement, but I guess we need more infra to fix it.